src/rust/graph-merger/src/service.rs

```diff
@@ -58,8 +56,8 @@ pub enum GraphMergerError {
     #[error("failed to configure tracing {0}")]
     SetupTracingError(#[from] SetupTracingError),
 
-    #[error("GraphMutationClientError {0}")]
-    GraphMutationClientError(#[from] GraphMutationClientError),
+    #[error("gRPC client error {0}")]
+    GrpcClientError(#[from] ClientError),
```

Comment:
In this PR I see both `GrpcClientError(#[from] ClientError)` and `ClientError(#[from] ClientError)`, could be worth standardizing (but not a big deal)